recipes/lfortran/LICENSE

```diff
@@ -0,0 +1,17 @@
+Copyright 2019. Triad National Security, LLC. All rights reserved.
```

Comment:
Can you add the `LICENSE` file to `MANIFEST.in` so that sdist will contain the license?

Reply:
Yes. I have to do this upstream.

Reply:
Fixed in https://gitlab.com/lfortran/lfortran/merge_requests/228/. I'll wait if there are more things to fix before making a new upstream release.

Reply:
Can you also print `CMAKE_C_FLAGS` in addition to `CMAKE_C_FLAGS_RELEASE`?

Reply:
Can you remove this file now?

Reply:
Done.